### PR TITLE
Added teamwork section

### PR DESCRIPTION
**Pull request by Faberen**

### CODE REVIEW

File: docs/Images/teamwork.png
Comment (SkarpKat) [inline docs/Images/teamwork.png:1]: Don't know it the dot in the end should be there. In an activity diagram i believe that the end node should be the end.

**Review by SkarpKat — APPROVED** [review]

**Review by annahj1006 — CHANGES_REQUESTED** [review]
The teamwork png is a bit wierd, can we maybe make the lines sharp, so it matches our other diagrams. 

Also, if this is an activity diagram, the whole thing should start with the dot, and then end with the dot. The first dot represents the start of the activity, and the end dot represents the end of the activity. 

Report under project board - "They were, adding tags to cheeps...." should perhabs be a : rather than just the start of the sentence, since we're listing them :)
On line 165 "When reviewing the code a reviewer can request changes and then further work on the issue is requred" might be missing a pause in the sentence, since i had to read it twice to understand the meaning - might just be me who's stupid tho

File: docs/Report.md [hunk 1]
```diff
@@ -151,12 +151,21 @@ Before putting anything into the workflow actions, we create test manually to ru
...
 
-Briefly describe and illustrate the flow of activities that happen from the new creation of an issue (task description), over development, etc. until a feature is finally merged into the main branch of your repository.
+## Project Board
+![Fix.XX Project Board](Images/ProjectBoard.png){width=60%}
+<br>
+This figure shows the Project board of Chirp at the day of hand-in. We have four issues that haven't been implemented before the deadline. All the four issues are under the Todo section but none of them are in the project requirement. That is to say they were formulated under the *Wild Style* development section. They were, adding tags to cheeps, being able to cheep a meme, trending cheeps and a re-cheep feature. As can be seen from the project board the re-cheep feature had an assigned developer but wasn't completed in time as other requirements had to be met. One Wild style feature was implemented, a like button for the cheeps. Although the like implementation is missing some functionality. A user can't see which cheep they've like and they can like a cheep infinitely many times.
+
```
Comment (AlbinoLoaf) [inline docs/Report.md:161]: This figure shows the Project board of Chirp on the day of the hand-in. We have four issues that haven't been implemented before the deadline. All four issues are under the Todo section. None of them are in the project requirement. That is to say, they were formulated under the *Wildstyle* development section. They were, adding tags to cheeps, being able to cheep a meme, trending cheeps and a re-cheep feature. As can be seen from the project board the re-cheep feature had an assigned developer but wasn't completed in time as other requirements had to be met. One Wildstyle feature was implemented a like button on the Cheeps. Although the like implementation is missing some functionality. A user can't see which cheep they've liked and they can like, a Cheep infinitely many times.

File: docs/Report.md [hunk 2]
```diff
@@ -151,12 +151,21 @@ Before putting anything into the workflow actions, we create test manually to ru
...
+Another more simple *"issue-creation-flow"* was also used. If a developer found a bug with the application or other small adjustment were made an issue was created. This created good documentation for the other developers so all group members could understand why a pull request was made.
```
Comment (AlbinoLoaf) [inline docs/Report.md:167]: Another more simple *"issue-creation-flow"* was also used. If a developer found a bug within the application or other small adjustments were made. An issue was created. This created good documentation for the other developers so all group members could understand why a pull request was made.

File: docs/Report.md [hunk 3]
```diff
@@ -151,12 +151,21 @@ Before putting anything into the workflow actions, we create test manually to ru
...
+
+## Issue creation
+![Fig.XX Flow of issues](Images/teamwork.png){width=60%}
+<br>
+This illustration show the flow of our work-process when being met with new requirements for the project from the lectures. At first the new requirements are read and understood, then the group gathers and tries to formulate the tasks into small issues which ideally can be completed within a days worth of work. If a formulation gets accepted by the group it gets posted on the issue board on github. A developer assigns themselves to an issue to let the others know what they are working on. When the developer feels like they've implemented the feature adequately they commit and create a pull request. When a pull request is posted 2 reviewers from the group are needed to further merge it to main and deploy. When reviewing the code a reviewer can request changes and then further work on the issue is required. This process repeats until two reviewers accept the changes and then the code can be merged with main. <br>
```
Comment (AlbinoLoaf) [inline docs/Report.md:165]: This illustration shows the flow of our work process 

**when being met with new requirements for the project from the lectures.** -- Odd 

 At first, the new requirements are read and understood, and then the group gathers and tries to formulate the tasks into small issues which ideally can be completed within a day's worth of work. If a formulation gets accepted by the group it gets posted on the issue board on github. A developer assigns themselves to an issue to let others know what they are working on. When the developer feels like they've implemented the feature adequately they commit and create a pull request. When a pull request is posted 2 reviewers from the group are needed to further merge it to main and deploy. When reviewing the code a reviewer can request changes and then further work on the issue is required. This process repeats until two reviewers accept the changes and then the code can be merged with main. <br>

**Review by AlbinoLoaf — CHANGES_REQUESTED** [review]

File: docs/Report.md (hunk 1 above)
Comment (Faberen) [reply docs/Report.md:161]: Thanks

File: docs/Report.md (hunk 2 above)
Comment (Faberen) [reply docs/Report.md:167]: Thanks

File: docs/Report.md (hunk 3 above)
Comment (Faberen) [reply docs/Report.md:165]: Thanks

File: docs/Images/teamwork.png
Comment (Faberen) [reply docs/Images/teamwork.png:1]: This isn't a UML activity diagram it's just an illustration representing the flow as per Helges requirements. But yes I have made it with an acitvity diagram in mind. The dot signifies the end state and thats confusing so I'll remove it

**Comment by Faberen** [comment]
> The teamwork png is a bit wierd, can we maybe make the lines sharp, so it matches our other diagrams.
> 
> Also, if this is an activity diagram, the whole thing should start with the dot, and then end with the dot. The first dot represents the start of the activity, and the end dot represents the end of the activity.
> 
> Report under project board - "They were, adding tags to cheeps...." should perhabs be a : rather than just the start of the sentence, since we're listing them :) On line 165 "When reviewing the code a reviewer can request changes and then further work on the issue is requred" might be missing a pause in the sentence, since i had to read it twice to understand the meaning - might just be me who's stupid tho

I disagree with making the lines sharp but would like to hear what the rest of the group thinks

It's not a specific UML diagram but it has taken inspiration from uml activity diagram. I have removed the end state as per request of Oliver. Helge has asked for an illustration that explains how we handle the creation of issues but not a specific uml diagram. I can if you want add a start and end state

I personally think making it a list would break the reading flow, if you don't like it I think omitting them would perhaps be better since we can see it on the project board.

**Review by annahj1006 — APPROVED** [review]
Nice nice

**Review by SkarpKat — APPROVED** [review]

File: docs/Report.md [hunk 4]
```diff
@@ -151,12 +151,21 @@ Before putting anything into the workflow actions, we create test manually to ru
...
+
+## Issue creation
+![Fig.XX Flow of issues](Images/teamwork.png){width=60%}
+<br>
+This illustration shows the flow of our work process. At first, the new requirements are read and understood, and then the group gathers and tries to formulate the tasks into small issues which ideally can be completed within a day's worth of work. If a formulation gets accepted by the group it gets posted on the issue board on Github. A developer assigns themselves to an issue to let others know what they are working on. When the developer feels like they've implemented the feature adequately they commit and create a pull request. When a pull request is posted two reviewers from the group are needed to further merge it to main and deploy. When reviewing the code a reviewer can request changes and then further work on the issue is required. This process repeats until two reviewers accept the changes and then the code can be merged with main.
```
Comment (AlbinoLoaf) [inline docs/Report.md:165]: GitHub instead of Github

File: docs/Report.md [hunk 5]
```diff
@@ -151,12 +151,21 @@ Before putting anything into the workflow actions, we create test manually to ru
...
 
-Briefly describe and illustrate the flow of activities that happen from the new creation of an issue (task description), over development, etc. until a feature is finally merged into the main branch of your repository.
+## Project Board
+![Fix.XX Project Board](Images/ProjectBoard.png){width=60%}
+<br>
+This figure shows the Project board of Chirp on the day of the hand-in. We have four issues that haven't been implemented before the deadline. All four issues are under the Todo section. None of them are in the project requirement. That is to say, they were formulated under the *Wildstyle* development section. They were, adding tags to cheeps, being able to cheep a meme, trending cheeps and a re-cheep feature. As can be seen from the project board the re-cheep feature had an assigned developer but wasn't completed in time as other requirements had to be met. One *Wildstyle* feature was implemented a like button on the Cheeps. Although the like implementation is missing some functionality. A user can't see which cheep they've liked and they can like, a Cheep infinitely many times.
```
Comment (AlbinoLoaf) [inline docs/Report.md:160]: We have more than one project requirements I believe

**Review by AlbinoLoaf — CHANGES_REQUESTED** [review]

File: docs/Report.md (hunk 5 above)
Comment (Faberen) [reply docs/Report.md:160]: What do you mean?

Comment (AlbinoLoaf) [reply docs/Report.md:160]: None of them are in the project requirements plural requirements

**Review by AlbinoLoaf — APPROVED** [review]
Nice job, well written and great clerity

**Pull request merged**